libs/generator/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_library(block_generator STATIC generator.cpp)
+
+if(NOT Boost_FOUND)
+    find_package(Boost COMPONENTS REQUIRED filesystem log log_setup program_options thread system json)
+endif()
+target_link_libraries(block_generator PUBLIC zkEVMDataTypes ${Boost_LIBRARIES})
+target_include_directories(block_generator PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+
+install(TARGETS block_generator DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/generator.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/generator)
```

Comment:
Do we preserve paths here? It seems safer to preserve directory structure of headers therefore user can include this like:

```cpp
#include "zkevm_framework/generator/generator.h"
```

instead of 

```cpp
#include "generator.h"
```

Especially if we take into account that "generator.h" is a pretty common name.

Reply:
Done